ext/opentelemetry-ext-opencensusexporter/src/opentelemetry/ext/opencensusexporter/metrics_exporter/__init__.py

```diff
@@ -162,3 +180,11 @@ def get_collector_point(metric_record: MetricRecord) -> metrics_pb2.Point:
             )
         )
     return point
+
+
+def get_resource(metric_record: MetricRecord) -> resource_pb2.Resource:
+    resource_labels = metric_record.instrument.meter.resource.labels
+    return resource_pb2.Resource(
+        type=resource_labels.get("service.name"),
```

Comment:
This should follow [these conventions](https://github.com/census-instrumentation/opencensus-specs/blob/master/resource/StandardResources.md) (except send `container` for k8s, based on how exporters behave)

Reply:
I think you still need to set the correct resource type here.

OpenCensus itself doesn't recognize semantic conventions, it's a [concept of OpenTelemetry](https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/resource/semantic_conventions), e.g. if you send metrics to [OpenCensus Service](https://github.com/census-instrumentation/opencensus-service) it won't infer the resource type.

The change in the collector is designed to address the case `SDK -> [OTLP] -> OpenTelemetry Collector -> [OpenCensus] -> ...`, not for `SDK -> [OpenCensus gRPC] -> ...`.

To clarify, I think it will actually work for `SDK -> [OpenCensus gRPC] -> OpenTelemetry Collector -> ...` as well, but that's mostly a coincidence.

Reply:
Sounds good. I will follow your example in open-telemetry/opentelemetry-collector#1462 for mapping from OT -> OC resource type